### ISSUE TITLE
Inso CLI .exe not doing anything and therefore cannot install

### ISSUE THREAD

**Issue by cwikholm** [getinsomnia/insomnia#6486]
### Expected Behavior

I expected to download Inso CLI, extract the executable, and then run the installer.

### Actual Behavior

Once I extract the executable, and attempt to launch the application nothing happens:

Extracted executable:
![image](https://github.com/Kong/insomnia/assets/128405004/8a951e35-1b1c-4c03-827c-35c5d6660e76)

After I click it, a command prompt window flashes for about 1 second, and then it closes and does nothing:
![image](https://github.com/Kong/insomnia/assets/128405004/3e18c8ac-e094-4fbf-908e-36318be46a29)

### Reproduction Steps

1.) Go to https://github.com/Kong/insomnia/releases/tag/lib@2023.5.8 to download the Windows .zip
2.) Extract the files using Windows explorer / file explorer
3.) Attempt to launch the application
4.) Click "Run anyway" on the security warning

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

inso-windows-2023.5.8

### What operating system are you using?

Windows

### Operating System Version

Windows 11 Version 22H2 (OS BUILD 22621.2215)

### Installation method

Download from Insomnia

**Comment by cwikholm** [comment]
@jackkav you closed this as completed but I am still facing the same issues? 
If I'm doing something wrong please let me know, thank you.

**Comment by jackkav** [comment]
https://docs.insomnia.rest/inso-cli/install

**Comment by cwikholm** [comment]
Still the exact same issue when I attempt to manually extract the executable and then run it.

And if I try the command, from the link you provided, this is what happens in the command prompt:

![image](https://github.com/Kong/insomnia/assets/128405004/a43e227c-c707-463e-93b8-54793d525658)

**Comment by lcsmobileqa** [comment]
I'm not sure if you've resolved this yet. I had this error come up as well and found a workaround in that I had to cd into the folder that held the executable. But then I ran into instructions online to add the folder containing the executable to my System environment variables path and that error stopped and the cli started doing what's expected. 
Another thing is that the '.' error also depended on which terminal I used, before setting the path. 
If I was in a powershell terminal, it required the ./inso, but if I was in the command prompt, then I could just use "inso". So powershell would have needed "./inso --version" and the windows CMD used "inso --version". With the path set, I can just use "inso --version" in both terminals now.

Hope this helps!
